x-pack/test/cases_api_integration/common/plugins/security_solution/server/plugin.ts

```diff
@@ -125,5 +130,40 @@ export class FixturePlugin implements Plugin<void, void, FixtureSetupDeps, Fixtu
         },
       },
     });
+
+    features.registerKibanaFeature({
```

Comment:
This should be moved to `x-pack/test/cases_api_integration/common/plugins/cases/server/plugin.ts`